saleor/graphql/product/mutations/products.py

```diff
@@ -385,14 +408,26 @@ def perform_mutation(cls, _root, info, collection_id, products):
         collection = cls.get_node_or_error(
             info, collection_id, field="collection_id", only_type=Collection
         )
-        products = cls.get_nodes_or_error(products, "products", Product)
+        products = cls.get_nodes_or_error(
+            products,
+            "products",
+            Product,
+            qs=models.Product.objects.prefetch_related(
+                "attributes", "collections", "variants", "category"
```

Comment:
Please double check the `attributes` prefetch. I think that it misses some relations.

Reply:
Good catch, thanks. I will go thought all product_updated and product_variant_updated to add prefetch.